spacy/matcher/matcher.pyi

```diff
@@ -1,4 +1,16 @@
-from typing import Any, List, Dict, Tuple, Optional, Callable, Union, Iterator, Iterable
+from typing import (
+    Any,
+    List,
+    Dict,
+    Tuple,
+    Optional,
+    Callable,
+    Union,
+    Iterator,
+    Iterable,
+    overload,
+)
```

Comment:
In these cases we prefer to make the split ourselves instead of having `black` turn this into so many separate lines:
```suggestion
from typing import Any, List, Dict, Tuple, Optional, Callable, Union
from typing import Iterator, Iterable, overload
```